lib/csrf.js

```diff
@@ -65,7 +67,11 @@ class CSRF {
     createToken();
 
     if (verify.error) {
-      return ctx.error(verify);
+      try {
+        return ctx.error(verify);
+      } catch (e) {
+        console.log(e);
```

Comment:
This is the CSRF engine, probably not a good idea to have a `console.log` here. 

- `ctx.error` should catch its own errors and log it

Reply:
Actually, this is causing these two tests to fail:

```

  1) koa middleware csrf should return 500 for missing jwt:
      AssertionError: expected 200 to equal 500
      + expected - actual
      -200
      +500
      
      at fetch.then.res (test/spec/csrf-koa-test.js:107:31)
      at <anonymous>
      at process._tickDomainCallback (internal/process/next_tick.js:228:7)
  2) koa middleware csrf should return 500 for invalid jwt:
      AssertionError: expected 200 to equal 500
      + expected - actual
      -200
      +500
      
      at fetch.then.res (test/spec/csrf-koa-test.js:126:33)
      at <anonymous>
      at process._tickDomainCallback (internal/process/next_tick.js:228:7)
```